modules/installing-ove-console-final.adoc

```diff
@@ -0,0 +1,78 @@
+====
+
+.Procedure
+
+. On the installation console hosted by the rendezvous node, configure the hosts in the *Host discovery* page:
+
+.. Verify that every machine you booted from the ISO image appears in the *Host Inventory* section and has a *Status* value of *Ready*.
+
+.. For each host, click the expand icon and verify that all of the specification fields are correct.
+
+.. Optional: For each host except the rendezvous node, configure the role by selecting an option from the dropdown menu of the *Role* column.
+The default value for every host except the rendezvous node is `Auto-assign`.
+
+.. Click *Next* to continue.
+
+. On the *Storage* page, click the expand icon for each host and verify that all of the specification fields are correct.
+
+. Click *Next* to continue.
+
+. If you will manage your own networking, select the *User-Managed Networking* option on the *Networking* page.
+
+. If the cluster will manage networking, select the *Cluster-Managed Networking* option on the *Networking* page and configure cluster networking:
+
+.. Select a *Networking stack type*.
+
+.. Optional: Select a machine network from the dropdown menu of the *Machine network* field.
+Otherwise, a default value is selected.
+
+.. Enter an IP address in the *API IP* field.
+An API IP provides an endpoint for all users to interact with and configure the platform.
+
+.. Enter an IP address in the *Ingress IP* field.
+An ingress IP provides an endpoint for application traffic flowing from outside the cluster.
+
+. Optional: Select the *Use advanced networking* checkbox and configure other parameters such as the *Cluster network CIDR*, the *Cluster network host prefix*, or *the Service network CIDR*.
++
+This option is available for both cluster-managed and user-managed networking.
+
+. Optional: Enter a key in the *Host SSH Public Key for troubleshooting after installation* field, which you can use to connect to hosts using a public SSH key for troubleshooting after installation.
++
+This option is available for both cluster-managed and user-managed networking.
+
+. Click *Next* to continue.
+
+. On the *Download credentials* page, select the checkbox to acknowledge that you must download credential files prior to cluster installation.
+
+. Click *Download credentials* and save the cluster credentials file in a secure location.
++
+[IMPORTANT]
+====
+You must download the credentials at this stage.
+Once you initiate the cluster installation, the rendezvous node reboots and you can no longer retrieve the credentials.
+====
+
+. On the *Review and create* page, review all of the cluster details and click *Install cluster* to initiate the cluster installation.
+
+During the installation process, the rendezvous node reboots and the console you used to configure the installation is no longer accessible.
```

Comment:
Same about making this DITA compliant. Info after the procedure steps have completed won't be supported in DITA. Consider adjusting that now as necessary to avoid having to come back and fix later